src/app/products/details/details.component.ts

```diff
@@ -16,37 +14,34 @@ import { LoaderComponent } from '../../loader/loader.component';
   standalone: true,
   templateUrl: './details.component.html',
   styleUrl: './details.component.scss',
-  imports: [CommonModule, FooterComponent, RouterLink, NavbarComponent,LoaderComponent],
+  imports: [
+    CommonModule,
+    FooterComponent,
+    RouterLink,
+    NavbarComponent,
+    LoaderComponent,
+  ],
 })
-export class DetailsComponent {
-  selectedProduct = this.store.selectSignal(selectedProduct);
-  loading =this.store.selectSignal(isProductsLoading);
-  selectedSize: string = '';
-  id!: string;
+export class DetailsComponent implements OnInit {
+  protected selectedProduct = this.store.selectSignal(selectedProduct);
+  protected loading = this.store.selectSignal(isProductsLoading);
+  protected id!: string;
```

Comment:
I don't think creating this property is necessary. I don't think you are using it anywhere else

Reply:
Which of the properties?
I am using it